RELEASE_NOTES.md

```diff
@@ -10,6 +10,108 @@
 Version vxx.yy.zz -- Release date 20yy-mm-dd
 ============================================
 
+Version v0.12.0 -- Release date 2020-09-03
+============================================
+
+This release contains new features, notable changes, and bug fixes. Please
+see the RAJA user guide for more information about items in this release.
+
+Notable changes include:
+
+  * Notable repository change:
+      * The 'master' branch in the RAJA git repo has been renamed to 'main'.
+
+  * New features:
```

Comment:
@rhornung67 @MrBurmark Did we want to make users aware of "sort" as a New Feature? Maybe a change like the following (just the first sentence from the sort documentation):
```suggestion
      * New RAJA "sort" capability added. This provides portable parallel sort operations, which are basic parallel algorithm building blocks.
```

Reply:
I thought I had it in there. Sorry. It is now added.